src/service/storage_arango.py

```diff
@@ -27,23 +27,6 @@
 ARANGO_ERR_NAME_EXISTS = 1207
 _ARANGO_ERR_UNIQUE_CONSTRAINT = 1210
 
-_QUERY_GET_NEXT_VERSION = f"""
```

Comment:
Easier to read the methods if the queries are inline. If the same aql is used for multiple methods then a module global might make sense